tests/unit/models/builder-test.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
-var fs              = require('fs-extra');
+var fs              = require('fs');
+var fse             = require('fs-extra');
```

Comment:
Why do we need both of these? I was pretty sure `fse` proxied to `fs`? Does `FSMonitor` not play nicely with the `fse` abstraction layer when being reset?